docs/overview/getting-started/avalanche-consensus.md

```diff
@@ -41,8 +41,9 @@ acceptance of non-conflicting transactions and the rejection of conflicting tran
 
 ![How Avalanche Consensus Works](/img/howavalancheconsensusworks.png)
 
-It is guaranteed (with high probability based on system parameters) that if any honest validator
-accepts or rejects a transaction, all honest validators will accept or reject that transaction.
+Avalanche Consensus guarantees (with high probability based on system parameters) that if any honest
+validator accepts or rejects a transaction, all honest validators will accept or reject that
```

Comment:
it might be more clear to say `if any honest validator accepts or rejects a transaction, all honest validators will come to the same conclusion.`

Reply:
Reworded